LICENSE-MIT

```diff
@@ -0,0 +1,24 @@
+Permission is hereby granted, free of charge, to any
```

Comment:
Didn't we drop this license in favor of BSD-3-Clause?

Reply:
Yes, I will remove this licence